KNOWN_ISSUES.md

```diff
@@ -26,21 +22,18 @@ Please report any other issue you may encounter by using the app help form.
 
 **Issues** caused by lower-level bugs that will be **harder to fix or circumvent**:
 
-* Replacing a directory with a file of the same name won't work (same when
-  replacing a file with a directory of the same name).
-  [details](https://trello.com/c/rBQ2XXwp)
-* When moving 2 directories at the same time on Windows, possibly only 1 move
-  may be detected. [details](https://trello.com/c/X3QMUQO3)
-
+* Files and directories at the root of the synchronized directory (usually
+  `Cozy Drive`) and whose name start with an underscore (i.e. `_`), won't be
+  synchronized at all.
+* On MacOS, Files in directories whose name start or end with a space won't be
+  synchronized although the directories themselves will be.
 
 ## Performances & resources consumption
 
-* The app takes some time before actually syncing, may hold on until
-  there is no more activity, and uses a lot of CPU on start and when syncing.
-  [details](https://trello.com/c/IQEImXQF)
+* The app uses a lot of CPU on start and when syncing.
```

Comment:
Isn't it still the case with chokidar?

Reply:
Oh, didn't see you moved it to the macOS section. My bad :)

Reply:
I moved it to the `MacOS` section.